src/modules/numbers/dto/get-numbers-query.dto.ts

```diff
@@ -5,12 +5,12 @@ export class GetNumbersQueryDto {
   @IsInt()
   @IsNotEmpty()
   @Max(9)
-  @ApiProperty({ example: 1 })
+  @ApiProperty({ example: 1, description: 'Id of UKEF ID type. Common types are: 1 for Deal/Facility, 2 for Party, 8 for Covenant' })
```

Comment:
Should we only be accepting `1, 2 or 8`?

Reply:
Can we also add `@Min(2)`, or a number you think is a common min standard please?

Reply:
Added @Min(1). 
We could limit accepted values in future, at the moment passing 3,4,5,6,7 allows testing and using logic in SP.